Build fails on qt 5.14 with gcc 9 on ZoomTool meta type

```
/usr/bin/ld: main.o: in function `int qmlRegisterType<QniteZoomTool>(char const*, int, int, char const*)':
/usr/include/qt/QtQml/qqml.h:293: undefined reference to `QniteZoomTool::staticMetaObject'
/usr/bin/ld: /usr/include/qt/QtQml/qqml.h:295: undefined reference to `QniteZoomTool::staticMetaObject'
/usr/bin/ld: main.o: in function `QQmlPrivate::QQmlElement<QniteZoomTool>::QQmlElement()':
/usr/include/qt/QtQml/qqmlprivate.h:106: undefined reference to `QniteZoomTool::QniteZoomTool(QQuickItem*)'
/usr/bin/ld: main.o: in function `QtPrivate::MetaObjectForType<QniteZoomTool*, void>::value()':
/usr/include/qt/QtCore/qmetatype.h:1556: undefined reference to `QniteZoomTool::staticMetaObject'
/usr/bin/ld: main.o: in function `QMetaTypeIdQObject<QniteZoomTool*, 8>::qt_metatype_id()':
/usr/include/qt/QtCore/qmetatype.h:1917: undefined reference to `QniteZoomTool::staticMetaObject'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0x10): undefined reference to `QniteZoomTool::metaObject() const'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0x18): undefined reference to `QniteZoomTool::qt_metacast(char const*)'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0x20): undefined reference to `QniteZoomTool::qt_metacall(QMetaObject::Call, int, void**)'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0xe0): undefined reference to `QniteZoomTool::mousePressEvent(QMouseEvent*)'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0xe8): undefined reference to `QniteZoomTool::mouseMoveEvent(QMouseEvent*)'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0xf0): undefined reference to `QniteZoomTool::mouseReleaseEvent(QMouseEvent*)'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0x190): undefined reference to `QniteZoomTool::createItemPainter() const'
/usr/bin/ld: main.o:(.data.rel.ro._ZTIN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTIN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0x10): undefined reference to `typeinfo for QniteZoomTool'
/usr/bin/ld: main.o: in function `QQmlPrivate::QQmlElement<QniteZoomTool>::~QQmlElement()':
/usr/include/qt/QtQml/qqmlprivate.h:111: undefined reference to `QniteZoomTool::~QniteZoomTool()'
collect2: error: ld returned 1 exit status
make[1]: Leaving directory 'build-qnite_examples-Desktop-Debug/examples/line'
make[1]: *** [Makefile:293: line] Error 1
make: *** [Makefile:48: sub-examples-line-make_first] Error 2
make: *** Waiting for unfinished jobs....
g++  -o plots qnanopainter.o qnanocolor.o qnanolineargradient.o qnanoimagepattern.o qnanoimage.o qnanofont.o qnanoradialgradient.o qnanoboxgradient.o qnanowindow.o qnanodebug.o qnanoquickitem.o qnanoquickitempainter.o qnanobackendgl2.o qnanobackendgl3.o nanovg.o qniteartist.o qnitexyartist.o qniteaxis.o qnitelinearaxis.o qnitecategoryaxis.o qniteaxistick.o qniteaxes.o qniteclipper.o qnitemapper.o qnitelinearmapper.o qniteticker.o qnitelinearticker.o qnitebar.o qnitebarpainter.o qnitecircle.o qnitecirclepainter.o qnitegrid.o qnitegridpainter.o qniteline.o qnitelinepainter.o qnitepen.o qnitespline.o qnitetool.o qniteselectiontool.o qnitepointselectiontool.o qnitepathselectiontool.o qnitepathpainter.o main.o qrc_libqnanopainterdata.o qrc_qnite.o qrc_plots.o moc_qnanowindow.o moc_qnanoquickitem.o moc_qniteartist.o moc_qnitexyartist.o moc_qniteaxis.o moc_qnitecategoryaxis.o moc_qniteaxistick.o moc_qniteaxes.o moc_qniteclipper.o moc_qnitemapper.o moc_qnitelinearmapper.o moc_qniteticker.o moc_qnitelinearticker.o moc_qnitebar.o moc_qnitecircle.o moc_qniteline.o moc_qnitegrid.o moc_qnitespline.o moc_qnitepen.o moc_qnitetool.o moc_qniteselectiontool.o moc_qnitepointselectiontool.o moc_qnitepathselectiontool.o   /usr/lib/libQt5Quick.so /usr/lib/libQt5Gui.so /usr/lib/libQt5QmlModels.so /usr/lib/libQt5Qml.so /usr/lib/libQt5Network.so /usr/lib/libQt5Core.so -lGL -lpthread   
/usr/bin/ld: main.o: in function `int qmlRegisterType<QniteZoomTool>(char const*, int, int, char const*)':
/usr/include/qt/QtQml/qqml.h:293: undefined reference to `QniteZoomTool::staticMetaObject'
/usr/bin/ld: /usr/include/qt/QtQml/qqml.h:295: undefined reference to `QniteZoomTool::staticMetaObject'
/usr/bin/ld: main.o: in function `QQmlPrivate::QQmlElement<QniteZoomTool>::QQmlElement()':
/usr/include/qt/QtQml/qqmlprivate.h:106: undefined reference to `QniteZoomTool::QniteZoomTool(QQuickItem*)'
/usr/bin/ld: main.o: in function `QtPrivate::MetaObjectForType<QniteZoomTool*, void>::value()':
/usr/include/qt/QtCore/qmetatype.h:1556: undefined reference to `QniteZoomTool::staticMetaObject'
/usr/bin/ld: main.o: in function `QMetaTypeIdQObject<QniteZoomTool*, 8>::qt_metatype_id()':
/usr/include/qt/QtCore/qmetatype.h:1917: undefined reference to `QniteZoomTool::staticMetaObject'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0x10): undefined reference to `QniteZoomTool::metaObject() const'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0x18): undefined reference to `QniteZoomTool::qt_metacast(char const*)'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0x20): undefined reference to `QniteZoomTool::qt_metacall(QMetaObject::Call, int, void**)'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0xe0): undefined reference to `QniteZoomTool::mousePressEvent(QMouseEvent*)'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0xe8): undefined reference to `QniteZoomTool::mouseMoveEvent(QMouseEvent*)'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0xf0): undefined reference to `QniteZoomTool::mouseReleaseEvent(QMouseEvent*)'
/usr/bin/ld: main.o:(.data.rel.ro._ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTVN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0x190): undefined reference to `QniteZoomTool::createItemPainter() const'
/usr/bin/ld: main.o:(.data.rel.ro._ZTIN11QQmlPrivate11QQmlElementI13QniteZoomToolEE[_ZTIN11QQmlPrivate11QQmlElementI13QniteZoomToolEE]+0x10): undefined reference to `typeinfo for QniteZoomTool'
/usr/bin/ld: main.o: in function `QQmlPrivate::QQmlElement<QniteZoomTool>::~QQmlElement()':
/usr/include/qt/QtQml/qqmlprivate.h:111: undefined reference to `QniteZoomTool::~QniteZoomTool()'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:293: plots] Error 1
```

Archlinux
Qt 5.14.2-1
gcc 9.3.0-1

Any Idea why the linking fails? Changes to the meta system?

--- a/qnite.prf
+++ b/qnite.prf
@@ -38,6 +38,8 @@ SOURCES += $$PWD/src/qniteartist.cpp \
         $$PWD/src/tools/qnitepointselectiontool.cpp \
         $$PWD/src/tools/qnitepathselectiontool.cpp \
         $$PWD/src/tools/qnitepathpainter.cpp \
+        $$PWD/src/tools/qnitezoomtool.cpp \
+        $$PWD/src/tools/qnitezoompainter.cpp \
 
 HEADERS += $$PWD/src/qnite.h \
         $$PWD/src/qniteartist.h \
@@ -67,6 +69,8 @@ HEADERS += $$PWD/src/qnite.h \
         $$PWD/src/tools/qnitepointselectiontool.h \
         $$PWD/src/tools/qnitepathselectiontool.h \
         $$PWD/src/tools/qnitepathpainter.h \
+        $$PWD/src/tools/qnitezoomtool.h \
+        $$PWD/src/tools/qnitezoompainter.h \
 
 RESOURCES += $$PWD/src/qnite.qrc
 

